packages/profiling-node/CHANGES

```diff
@@ -0,0 +1,500 @@
+## 1.3.3
```

Comment:
Do we want to keep this or should I remove the file entirely?

Reply:
Let's keep it - but add a note at the top saying this is an old changelog, and newer items will go in the main `CHANGELOG.md`